action.yml

```diff
@@ -0,0 +1,28 @@
+name: 'File Existence and Reader'
+description: 'Check existence and read'
+branding:
```

Comment:
Where does this get reflected?
Please verify with Pulkit wherever PM inputs are required.